docs/framework/guides/contributing/git-commit-message-convention.md

```diff
@@ -43,25 +43,69 @@ BREAKING CHANGE (package-name): Another breaking change if needed. Closes #ZZZ.
 
 Each commit can contain additional notes that will be inserted into the changelog:
 
-* `MAJOR BREAKING CHANGE` (alias: `BREAKING CHANGE`),
+* `MAJOR BREAKING CHANGE`,
 * `MINOR BREAKING CHANGE`.
 
 If any change contains the `MAJOR BREAKING CHANGE` note, the next release will be marked as `major` automatically.
 
 For reference on how to identify minor or major breaking change see the {@link framework/guides/support/versioning-policy versioning policy guide}.
 
-Each `BREAKING CHANGE` note must be followed by the package name.
+Each `MAJOR BREAKING CHANGE` or `MINOR BREAKING CHANGE` note must be followed by the package name.
```

Comment:
I think we could remove this rule. Any major BC causes the entire project to have a major release - hence, it's ok if we don't specify which package is affected (e.g. in case of DLLs). Same for minor - the entire project is bumped to the same version.

cc @pomek Do we need to change anything in the changelog generator?

Reply:
https://github.com/ckeditor/ckeditor5-dev/blob/f60d89c17bbc3ab358ce5a103e5851ff68285f4c/packages/ckeditor5-dev-env/lib/release-tools/tasks/generatechangelogformonorepository.js#L483-L562

Reply:
So what's the status for now? We leave the rule and the docs as-is? Or do we have any follow-up task for removing the rule?